src/dotnet/APIView/ClientSPA/src/app/_components/api-revision-options/api-revision-options.component.ts

```diff
@@ -124,15 +124,14 @@ export class ApiRevisionOptionsComponent implements OnChanges {
     });
 
     if (dropDownMenu === "active") {
-      this.activeApiRevisionsMenu = filtered;
+      this.activeApiRevisionsMenu = filtered.filter((apiRevision: APIRevision) => apiRevision.id !== this.diffApiRevisionId);
```

Comment:
to avoid the duplication, consider doing it before the `if` statements, like u did in line 122

Reply:
Am not sure what you  mean by duplication.